src/main/java/at/hannibal2/skyhanni/data/IslandGraphs.kt

```diff
@@ -150,8 +147,7 @@ object IslandGraphs {
     private var fastestPath: Graph? = null
     private var condition: () -> Boolean = { true }
     private var inGlaciteTunnels: Boolean? = null
-    private var nextChatMessage: ChatComponentText? = null
-    private var lastMessageSent = SimpleTimeMark.farPast()
+    private var nextMessageComponent: ChatComponentText? = null
```

Comment:
This is now entirely useless. Just call the new function/class directly when currently creating this helper object and do nothing on tick evenr